code/game/antagonist/antagonist.dm

```diff
@@ -86,8 +86,8 @@
 		Think through your actions and make the roleplay immersive! <b>Please remember all \
 		rules aside from those without explicit exceptions apply to antagonists.</b>"
 	
-	// Map template that antag needs to load before spawning. Nulled after it's loaded.
-	var/datum/map_template/base_to_load
+	// Map template name that antag needs to load before spawning. Nulled after it's loaded.
+	var/base_to_load
```

Comment:
Not loving this change; paths are checked (for existence at least) at compile time, here you can make a typo and have it fail.

Reply:
The point of this change is to allow moving templates off paths and into JSON.